src/main/java/hudson/plugins/parameterizedtrigger/BuildTrigger.java

```diff
@@ -96,7 +98,16 @@ public void buildDependencyGraph(AbstractProject owner, DependencyGraph graph) {
 		for (BuildTriggerConfig config : configs) {
 			List<AbstractProject> projectList = config.getProjectList(owner.getParent(), null);
 			for (AbstractProject project : projectList) {
-				ParameterizedDependency.add(owner, project, config, graph);
+                            if (config.triggerFromChildProjects() && owner instanceof ItemGroup) {
+                                ItemGroup parent = (ItemGroup) owner;
+                                for (AbstractProject child : (Collection<AbstractProject>)parent.getItems()) {
```

Comment:
This cast is strictly speaking unsafe as there is no guarantee it contains `AbstractProject`s.